torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -636,33 +636,52 @@ graph(%self, %input, %batchnorm):
     %second_output = aten::relu_(%first_output)
     return (%second_output) )", {is_batchnorm2d_module});
 
-
   const PatternInfo mul_nn_relu = PatternInfo::parse_from_str(R"(
-graph(%self, %a, %b):
+graph(%self, %a, %b, %relu):
```

Comment:
more for my own understanding, this is not a logic change and just moving `%relu` out, right?

Reply:
this is a logic change, we are improving the support here to be able to match relu modules used multiple times and the matching in `is_relu_module` is also more accurate than `match::module["ReLU"]`